packages/frontend/src/features/changesets/components/ProjectChangesets.tsx

```diff
@@ -0,0 +1,423 @@
+                                        ref={isTruncated.ref}
+                                        style={{ flex: 1 }}
+                                    >
+                                        {changeValue}
+                                    </Text>
+                                </Tooltip>
+                            </Group>
+                        );
+                    },
+                },
+                {
+                    accessorKey: 'createdByUserUuid',
+                    header: 'Created By',
+                    enableSorting: false,
+                    size: 200,
+                    Cell: ({ row }) => {
+                        const change = row.original;
+                        const userName = getUserDisplayName(
+                            change.createdByUserUuid,
+                            organizationUsers,
+                        );
+
+                        return (
+                            <Group gap="xs" align="center">
+                                <Tooltip
+                                    withinPortal
+                                    label={formatDateTime(change.createdAt)}
+                                >
+                                    <Box style={{ cursor: 'help' }}>
+                                        <MantineIcon
+                                            icon={IconClock}
+                                            size={16}
+                                            color="gray.6"
+                                        />
+                                    </Box>
+                                </Tooltip>
+                                <Text fz="sm" truncate>
+                                    {userName}
+                                </Text>
+                            </Group>
+                        );
+                    },
+                },
+            ],
+            [organizationUsers],
+        );
+
+    const table = useMantineReactTable({
+        columns,
+        data: allChanges,
+        enableColumnResizing: false,
+        enableRowNumbers: false,
+        enableRowVirtualization: false,
+        enablePagination: false,
+        enableFilters: false,
+        enableFullScreenToggle: false,
+        enableDensityToggle: false,
+        enableColumnActions: false,
+        enableColumnFilters: false,
+        enableHiding: false,
+        enableGlobalFilterModes: false,
+        enableSorting: false,
+        enableTopToolbar: false,
+        mantinePaperProps: {
+            shadow: undefined,
+            sx: {
+                border: `1px solid ${theme.colors.gray[2]}`,
+                borderRadius: theme.spacing.sm,
+                boxShadow: theme.shadows.subtle,
+                display: 'flex',
+                flexDirection: 'column',
+            },
+        },
+        mantineTableContainerProps: {
+            sx: {
+                maxHeight: 'calc(100dvh - 350px)',
```

Comment:
350 px is scary

Reply:
also enabled sticky header - don't know why it's not the default 😠